src/pages/organisationUnitGroupSets/form/OrganisationalUnitGroupSetFormFields.tsx

```diff
@@ -0,0 +1,105 @@
+import i18n from '@dhis2/d2-i18n'
+import { CheckboxFieldFF, RadioFieldFF } from '@dhis2/ui'
+import React from 'react'
+import { Field as FieldRFF, Field } from 'react-final-form'
+import {
+    CustomAttributesSection,
+    DefaultIdentifiableFields,
+    StandardFormField,
+    StandardFormSection,
+    StandardFormSectionTitle,
+    StandardFormSectionDescription,
+    DescriptionField,
+    HorizontalFieldGroup,
+    ModelTransferField,
+} from '../../../components'
+import { SECTIONS_MAP } from '../../../lib'
+import { DataDimensionField } from '../../dataElementGroupSets/fields'
+
+export const OrganisationalUnitGroupSetFormFields = () => {
+    const section = SECTIONS_MAP.category
+
+    return (
+        <>
+            <StandardFormSection>
+                <StandardFormSectionTitle>
+                    {i18n.t('Basic information')}
+                </StandardFormSectionTitle>
+                <StandardFormSectionDescription>
+                    {i18n.t(
+                        'Set up the basic information for this organisational unit group set.'
+                    )}
+                </StandardFormSectionDescription>
+                <DefaultIdentifiableFields />
+                <StandardFormField>
+                    <DescriptionField schemaSection={section} />
+                </StandardFormField>
+                <StandardFormField>
+                    <FieldRFF
+                        component={CheckboxFieldFF}
```

Comment:
I noticed that it seems like the label/clickable area for `CheckboxField` spans the entire width of the form. This is of course not something introduced in this PR. But may be something we want to take a look at? The easiest solution is to add `display: flex` to `StandardFormField` , I think? But we've also talked about removing the need for the `StandardFormField`-wrappers, and instead add gap to `StandardFormSection`.